modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/io/FileInput.java

```diff
@@ -102,7 +111,7 @@ public Crc32CheckingFileInput(FileInput delegate, boolean skipCheck) {
                 ensure(5);
 
                 throw new IgniteDataIntegrityViolationException(
-                    "val: " + val + " writtenCrc: " + writtenCrc
+                    "val: " + val + ", writtenCrc: " + writtenCrc + ", crcStartPos: " + calcStartPos + ", crcEndPos: " + calcEndPos
```

Comment:
In the ticket description three values were mentioned: WAL record size, CRC start offset, CRC end offset. But only two were added. Is this planned?

Reply:
It's not that simple right here, because abstraction level is lower than where WAL records are. The point is to see the range on which CRC is calculated - crcStartPos and crcStartPos should let us know.